packages/date-picker/src/FilterDateRangePicker/components/DateRangeValidationMessage/DateRangeValidationMessage.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react"
+import { FieldMessage } from "@kaizen/draft-form"
+import {
+  DateRangeValidationMessageType,
+  DateRangeValidationStatus,
+} from "../../../types"
+import styles from "./DateRangeValidationMessage.module.scss"
+
+export type DateRangeValidationMessageProps = {
+  /**
+   * Updates the styling of the validation FieldMessage
+   */
+  status?: DateRangeValidationStatus
+  /**
+   * A descriptive message for `status` states
+   */
+  validationMessage?: DateRangeValidationMessageType
+  id?: string
+  isReversed?: boolean
+}
+
+export const DateRangeValidationMessage = ({
+  status,
+  validationMessage,
+  id,
+  isReversed,
+}: DateRangeValidationMessageProps): JSX.Element | null => {
+  if (!validationMessage) return null
+
+  if (status?.dateEnd !== undefined && status?.dateStart !== undefined) {
```

Comment:
So there's no way to have a single message but have both fields have an error border?

Reply:
I think that is not a pattern we actually want to encourage - There should be a message for each to provide enough feedback to the user